config/game_options.txt

```diff
@@ -214,11 +214,20 @@ NEAR_DEATH_EXPERIENCE
 ## ROUNDSTART SILICON LAWS ###
 ## This controls what the AI's laws are at the start of the round.
 ## Set to 0/commented out for "off", silicons will just start with Asimov.
-## Set to 1 for "custom", silicons will start with the custom laws defined in silicon_laws.txt. (If silicon_laws.txt is empty, the AI will spawn with asimov and Custom boards will auto-delete.)
+## Set to 1 for "specified", silicons will start with an existing lawset. (If no specified lawset is identified, the AI will spawn with asimov.)
 ## Set to 2 for "random", silicons will start with a random lawset picked from random laws specified below.
 ## Set to 3 for "weighted random", using values in "silicon_weights.txt", a law will be selected, with weights specifed in that file.
+## Set to 4 for "custom", silicons will start with the custom laws defined in silicon_laws.txt. (If silicon_laws.txt is empty, the AI will spawn with asimov and Custom boards will auto-delete.)
```

Comment:
I feel like this unnecessarily breaks old configs. if custom was left at 1, and specified was 4, this wouldn't break existing configs that specified 1 to mean custom.

Reply:
If you did want to change this to 4 for custom. you need to rename the config and add a depreciation handler to convert the value from the old name to the new config.

Reply:
Fixes in https://github.com/tgstation/tgstation/pull/68587/commits/542807c5e5a5d139685fa0ef5a7b990eeb91677d